vm-virtio/src/rng.rs

```diff
@@ -141,6 +152,10 @@ impl RngEpollHandler {
                         debug!("KILL_EVENT received, stopping epoll loop");
                         break 'epoll;
                     }
+                    PAUSE_EVENT => {
+                        debug!("PAUSE_EVENT received, pausing epoll loop");
+                        thread::park();
```

Comment:
Aren't you supposed to run a loop to make sure you re-park it if it unparks for some reasons?

Reply:
Ah right, park could spuriously return. Let me fix that.